shephard-core/src/main/java/com/devsda/platform/shepherdcore/service/documentservice/ExecutionDocumentService.java

```diff
@@ -38,7 +44,7 @@ public boolean insertExecutionDetails(ExecuteWorkflowRequest executeWorkflowRequ
             try {
                 executionMetaDataJson= JSONLoader.stringify(metaData);
             } catch (IOException ex){
-                log.error(String.format("Unable to Process the initial payload for execution id : %s.", executeWorkflowRequest.getExecutionId()), ex);
+                log.error(String.format("Unable to stringify metadata %s, %s", metaData, ex), ex);
```

Comment:
no need to pass ex in first parameter. Add identifier in error message. objectId, executionId.